Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/AdditionalFields/AdditionalFieldForm.tsx

```diff
@@ -115,7 +115,7 @@ export default function AdditionalFieldForm(props: IProps) {
             e.push('A Field Name under 200 characters is required.');
         if (!Valid('ExternalDBTableID'))
             e.push('If an External Database is selected, then an External Database Table is required.');
-        if (props.Record.FieldName.match(/[$&+,:;=?@#|'<>.^*()%!{}`~]/) !== null)
+        if (props.Record.FieldName && props.Record.FieldName.match(/[$&+,:;=?@#|'<>.^*()%!{}`~]/) !== null)
```

Comment:
I don't think this is good, because it trips the error "Special characters are not allowed" when blank now. I'd recommend simply moving this up the error checking list and putting else on the valid('FieldName') condition instead, so we only check if the field name COULD be valid.